### DISCUSSION TITLE
Support Zen Mode in additional windows

### DISCUSSION THREAD

**Discussion by Friedolito** [oxdc/obsidian-vertical-tabs#72] [Ideas]
Hey thanks for developing this plug-in!

I love zen mode but currently it's only supported in the main window. 
I usually work with several windows at once and often I'm wondering why the shortcut for zen mode broke, 
until I realize that it is not the main window. :D 
Having zen mode in all windows would be neat! 

Best regards :)

**Comment by oxdc** [comment]
Hi @Friedolito ! Thank you for your suggestion. Currently, I don't know how to control these additional windows because there is no API available. I'll try to make this work though.

**Comment by Friedolito** [comment]
Thanks for the quick reply!
Nothing to major but happy if it would work. :)

**Comment by oxdc** [comment]
Hi @Friedolito ! I'll convert this issue to a discussion. Currently, there are some problems for implementing this feature. The most important one is UX. Because there is no sidebar in these additional windows, you may lose access to some tabs if *Vertical Tabs* is only visible in the main window, which may be another monitor.
